### PR TITLE
Move page objects into page objects folder

### PR DESCRIPTION
**Pull request by petterwildhagen**
Move page objects which are code interacting with specific elements or elements on specific pages into page_objects folder

### CODE REVIEW

File: frontend/cypress/support/common.ts [hunk 1]
```diff
@@ -95,6 +31,7 @@ Cypress.Commands.add('testCacheAndDB', (testCache: Function, testDB: Function =
     cy.reloadBmt()
     testDB()
 })
+export {}
```
Comment (achaikou) [inline frontend/cypress/support/common.ts:34]: What is this?

I can't find explanation in any language references :sweat_smile:

**Review by achaikou — APPROVED** [review]
I like that! :heart: 
Just probably squash the weird commits before merging.

File: frontend/cypress/support/common.ts (hunk 1 above)
Comment (petterwildhagen) [reply frontend/cypress/support/common.ts:34]: It says the module is exported which is necessary to make declare global work

Comment (petterwildhagen) [reply frontend/cypress/support/common.ts:34]: It indicates that the module is exported, which is necessary to make the declare global work

**Pull request merged**